rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] I think it would be better if `book and person` are an `attr_accessor` so then they can be read and written 🙃